clrdefinitions.cmake

```diff
@@ -35,6 +35,7 @@ elseif (CLR_CMAKE_TARGET_ARCH_ARM)
   add_definitions(-D_TARGET_ARM_=1)
   add_definitions(-DDBG_TARGET_32BIT=1)
   add_definitions(-DDBG_TARGET_ARM=1)
+  add_definitions(-DFEATURE_MULTIREG_RETURN)
```

Comment:
Why is defining `FEATURE_MULTIREG_RETURN` necessary? 
`FEATURE_MULTIREG_RETURN` is a VM setting, different from the JIT's `FEATURE_MULTIREG_RET`  switch.
I don't think the VM setting is relevant for HFAs